test/wrapeth/provider_test.exs

```diff
@@ -0,0 +1,103 @@
+defmodule Wrapeth.ProviderTest do
+  use ExUnit.Case, async: true
+  import Mox
+
+  setup do
+    {:ok, valid_address: "0x4Bd040d48fdD1C667fcE5fdDd326681766E6ad91"}
+  end
+
+  setup do
+    {:ok, invalid_address: "0y4Bd040d48fdD1C667fcE5fdDd326681766E6ad91"}
+  end
+
+  setup do
+    {:ok, invalid_block_number: "0y9710d"}
+  end
```

Comment:
Do you need this to be separated into different setups?

Reply:
It was a noob move, just fixed in the last commit.

Reply:
Not noob, but I had the idea that ExUnit documentation was more explicit about this.
You can also return this as a Keyword and don't need the tuple.